### PR TITLE
Updated world border release

### PR DESCRIPTION
**Pull request by TealNerd**
now actually uses world border instead of arbitrary number

### CODE REVIEW

**Comment by ttk2** [comment]
so it grabs the number from WB? Nice, but as far as random convenient features go auto turning off/on pearl feeding would be much much nicer. 

But thanks anyways, we will probably find this helpful.

**Comment by TealNerd** [comment]
What do you mean off and on pearl feeding?

**Comment by ttk2** [comment]
if we restart outside of schedule pearl feeding might be run, this could
cause pearls to be incorrectly freed, so we have to manually turn it off
for unscheduled restarts, we often forget to turn it back on for days at a
time (because we are restarting the server several times and never sure
when we will be done until we are), or worse we forget to turn it off in
the first place and have to repearl people.

**Comment by rourke750** [comment]
Looks good.  Its going to fuck with jenkins though...  @WildWeazel can you add a worldborder pom to jenkins or tell me how to do it.

**Comment by rourke750** [comment]
@WildWeazel So i got a wget into the lib-setup.  How do I get it to build the maven repos?

**Comment by TealNerd** [comment]
@rourke750 can you merge this so i can send a pull for the auto disable thing?

**Comment by rourke750** [comment]
@TealNerd You've already included the pull.

**Comment by TealNerd** [comment]
Oh yeah I didn't realize I did that lol

**Comment by ttk2** [comment]
what would merging this do to jenkins?

**Comment by WildWeazel** [comment]
I don't know what's happening. Why are we building WB? It's a soft runtime dependency.

**Comment by ttk2** [comment]
because its a dependency for PrisonPearl if we want to grab it.

**Comment by rourke750** [comment]
Can you show me how to get it to do the thing where we can add a reference
to it so it builds?

**Comment by WildWeazel** [comment]
It's not a build dependency though, at least according to this pull. It just asks Bukkit if it's loaded at runtime.

If you do need to build with it, just follow the Maven install targets for the other jars in lib-setup. It doesn't need its own pom.

**Comment by TealNerd** [comment]
It doesn't have to be installed to run the plugin but it is a dependency to build

**Comment by rourke750** [comment]
There are imports from wb in pp now because of that pull.  Jenkins will
fail at compiling with no reference.  I added the wb jar to lib set up but
I didn't see in the log do anything besides download it. Is that all it
needs to do?

**Comment by TealNerd** [comment]
It sounds like that's all it needs. Merge the pull and see if it builds?

**Comment by WildWeazel** [comment]
You have to install it to the local repo. That's what the all Maven steps after the download script are doing.

**Comment by rourke750** [comment]
If you look here: http://192.99.169.83:8080/job/lib-setup/28/console  it
downloads but it doesnt install.

On Wed, Apr 15, 2015 at 11:41 AM, Travis Christian <notifications@github.com

File: pom.xml [hunk 1]
```diff
@@ -54,6 +54,11 @@
       <version>1.0.1-11</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.wimbli.WorldBorder</groupId>
```
Comment (rourke750) [inline pom.xml]: change this to:
com.wimbli

File: pom.xml [hunk 2]
```diff
@@ -54,6 +54,11 @@
...
+      <artifactId>WorldBorder</artifactId>
+      <version>1.8.1</version>
```
Comment (rourke750) [inline pom.xml]: Change this to 1.8.3

**Comment by rourke750** [comment]
@TealNerd Make those changes and ill merge

**Comment by TealNerd** [comment]
Make which changes?

**Comment by rourke750** [comment]
Scroll up maybe.

**Comment by TealNerd** [comment]
Oh didn't see that, will do

**Comment by TealNerd** [comment]
actually @rourke750 it should be com.wimbli.WorldBorder because that's the main package. the main class is then com.wimbli.WorldBorder.WorldBorder

**Comment by rourke750** [comment]
@TealNerd How I've set it up, you need to do what I said

**Comment by TealNerd** [comment]
Yeah alright

**Comment by WildWeazel** [comment]
http://civcraft.co:8080/job/PrisonPearl-master/73/

**Comment by ttk2** [comment]
on civtest.

**Pull request merged**